Documentation/concepts/ebpf/lifeofapacket.rst

```diff
@@ -8,6 +8,15 @@
 Life of a Packet
 ################
 
+This documentation page overviews the life of a packet from an eBPF datapath perspective 
+by explaining three different scenarios. 
+
+.. admonition:: Video
+  :class: attention
+
+  You can also watch a video explanation of this page that also explores the tools available within Cilium to help 
+  you understand the life of a packet in `eCHO Episode 51: Life of a Packet with Cilium <https://youtu.be/0BKU6avwS98?t=494>`__.
```

Comment:
Please use the full youtube.com URL for consistency with the other links.

Reply:
Implemented the fix. Thank you